bridge/github/config.go

```diff
@@ -217,36 +387,42 @@ func promptURL() (string, string, error) {
 		}
 
 		line = strings.TrimRight(line, "\n")
-
 		if line == "" {
 			fmt.Println("URL is empty")
 			continue
 		}
 
-		projectUser, projectName, err := splitURL(line)
-
+		// get owner and project from url
+		_, owner, project, err := splitURL(line)
 		if err != nil {
 			fmt.Println(err)
 			continue
 		}
 
-		return projectUser, projectName, nil
+		return owner, project, nil
 	}
 }
 
-func splitURL(url string) (string, string, error) {
-	re, err := regexp.Compile(`github\.com\/([^\/]*)\/([^\/]*)`)
-	if err != nil {
-		panic(err)
+func splitURL(url string) (string, string, string, error) {
```

Comment:
With that much return values, it might be time to use golang's named return value